webapp/commonComponents/expression/expressionEditorPopup.js

```diff
@@ -74,12 +75,12 @@ class ExpressionEditorPopup extends React.Component {
         <div className="expression-editor-popup__footer">
           <button className="btn btn-xs btn-of"
                   onClick={() => onChange('')}
-                  aria-disabled={R.isEmpty(query)}>
-            <span className="icon icon-undo2 icon-16px"/> Reset
+                  aria-disabled={!Expression.toString(expr)}>
```

Comment:
why can't you use query here?

Reply:
Ok to the getSqlOperator but I think it's safer to throw an exception
Query can be null also if expr is null as depending on the parent, either query or expr are passed

Reply:
that was not my question.. this answers probably refers to another question